spec/support/factory_girl.rb

```diff
@@ -14,4 +14,14 @@
     is_public true
     user
   end
+
+  factory :adventure do
+    name 'What Bar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.